gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -4,184 +4,265 @@
 
 import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
 import NxAccordionSimpleExample from './NxAccordionExample';
 import NxAccordionComplexExample from './NxAccordionComplexExample';
 import NxAccordionTertiaryButtonExample from './NxAccordionTertiaryButtonExample';
+import NxStatefulAccordionExample from '../NxStatefulAccordion/NxStatefulAccordionExample';
 
 const NxAccordionSimpleCode = require('./NxAccordionExample?raw'),
     NxAccordionComplexCode = require('./NxAccordionComplexExample?raw'),
-    NxAccordionTertiaryButtonCode = require('./NxAccordionTertiaryButtonExample?raw');
+    NxAccordionTertiaryButtonCode = require('./NxAccordionTertiaryButtonExample?raw'),
+    NxStatefulAccordionCode = require('../NxStatefulAccordion/NxStatefulAccordionExample?raw');
 
-const NxAccordionPage = () =>
-  <>
-    <GalleryDescriptionTile>
-      <NxP>
-        NxAccordion renders a panel with an always-visible header and a collapsible/expandable body section.
-        This is analogous to the HTML 5 <NxCode>&lt;details&gt;</NxCode> element (which it is
-        implemented on top of). There are three related components: <NxCode>NxAccordion</NxCode> itself,
-        <NxCode>NxAccordion.Header</NxCode> which represents the header content, and
-        {' '}<NxCode>NxAccordion.Title</NxCode> which is a convenience component for the header title. All other
-        children of <NxCode>NxAccordion</NxCode> aside from
-        the <NxCode>Header</NxCode> and the <NxCode>Title</NxCode> are rendered in the collapsible section.
+export default function NxAccordionPage() {
+  const [activeTabId, setActiveTabId] = useState(0);
 
-        Note that this component is stateless – its open state must be tracked externally.
-        See <NxCode>NxStatefulAccordion</NxCode> for a version which tracks its own open state.
-      </NxP>
-      <NxTile.Subsection>
-        <NxTile.SubsectionHeader>
-          <NxH3>NxAccordion</NxH3>
-        </NxTile.SubsectionHeader>
-        <NxTable>
-          <NxTable.Head>
-            <NxTable.Row>
-              <NxTable.Cell>Name</NxTable.Cell>
-              <NxTable.Cell>Type</NxTable.Cell>
-              <NxTable.Cell>Required</NxTable.Cell>
-              <NxTable.Cell>Description</NxTable.Cell>
-            </NxTable.Row>
-          </NxTable.Head>
-          <NxTable.Body>
-            <NxTable.Row>
-              <NxTable.Cell>onToggle</NxTable.Cell>
-              <NxTable.Cell>(() =&gt; void)</NxTable.Cell>
-              <NxTable.Cell>No</NxTable.Cell>
-              <NxTable.Cell>
-                A function which gets called when the accordion collapse/expand state is toggled.
+  return (
+    <>
+      <NxTabs activeTab={activeTabId} onTabSelect={setActiveTabId}>
+        <NxTabList aria-label="Table Component Examples">
+          <NxTab>Usage</NxTab>
+          <NxTab>Stateless Examples</NxTab>
+          <NxTab>Stateful Examples</NxTab>
```

Comment:
Perhaps these should be called "React Stateless Examples" and "React Stateful Examples"? Considering in the `NxTable` page we have "React Examples" and "HTML Examples".

Reply:
> Perhaps these should be called "React Stateless Examples" and "React Stateful Examples"? Considering in the `NxTable` page we have "React Examples" and "HTML Examples".

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/607/commits/c5e04ef3037e9ad665ef83c6b03f0d625d92119f